ajax/libs/polyglot/package.json

```diff
@@ -1,10 +1,30 @@
 {
   "name": "polyglot",
   "filename": "js/jquery.polyglot.language.switcher.min.js",
-  "version": "2.2.0",
+  "version": "2.2",
```

Comment:
Why is this being changed? The files in this repo are still `2.2.0`.

Reply:
I note that the version tagged in the repo is `2.2` but as we have `2.2.0` here, there is nothing we can do. The bot should pick up any tagged versions going forward correctly.

Reply:
@MattIPv4 Look, we have already on cdnjs old lib that was added, as I think, manually. I have no idea why the version was changed and if the images folder is needed. IMO we have to remove version from package.json and do not add images folder

Reply:
Not adding images is fine, but we should not change the version here as it will break the website & API. Once a new version is pulled in, this will resolve itself.